schemas/HPXMLDataTypes.xsd

```diff
@@ -1663,6 +1664,13 @@
 			<xs:enumeration value="other"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="AtticWallType">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="gable"/>
+			<xs:enumeration value="knee wall"/>
+			<xs:enumeration value="other"/>
+		</xs:restriction>
```

Comment:
Noel, I like what you've done with the Attic.  Breaking out the Roof/Floor/Walls seems like a good way to go.

For the walls, would it help to indicate whether the wall is exposed to the environment or to a conditioned space?  My understanding is that the Gable would separate the attic interior from the outside, and a Kneewall would separate the attic interior from the bottom triangle of the roof, where the triangle could be insulated at the kneewall or at the slope/flat.  I'd assume the gable is exposed to the environment, but the kneewall may or may not be.  Does this matter?